### PR TITLE
Profile edit - part one

### LINKED ISSUE #217: Implement Linode detail settings Advanced Profile edit part un

### PR DESCRIPTION
**Pull request by SirCmpwn**
Closes #217 

Take particular note of fb3fc69 when reviewing

### CODE REVIEW

**Comment by eatonphil** [comment]
We (can't remember who this included) talked a while ago about specifically not having the cancel button in the top right corner because it's too far out of the field of vision. Other more viable options were to have it beside the Save button or beside the Edit profile section.

**Comment by SirCmpwn** [comment]
I'll move it next to the save button.

**Comment by SirCmpwn** [comment]
Added helpers and rearranged things a bit

**Comment by SirCmpwn** [comment]
The decreased coverage is just churn from increased line totals and existing gaps in coverage. Ready for merge.

**Comment by eatonphil** [comment]
Vote to move the Virtualization question mark to beside the Virtaulization label similar to how it is in the title.

File: src/linodes/settings/layouts/ConfigEdit.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+              return (
+                <option key={kernel.id} value={kernel.id}>
+                  {map[kernel.id] || kernel.label}
+                </option>);
+            })}</select>
+          </div>
+        </div>
+        <hr />
+        <fieldset className="form-group row">
+          <legend className="col-sm-2 col-form-legend">
```
Comment (eatonphil) [inline src/linodes/settings/layouts/ConfigEdit.js]: legend? What happened to labels?

Comment (SirCmpwn) [inline src/linodes/settings/layouts/ConfigEdit.js]: Bootstrap guildelines for horizontal forms wrt radio buttons and checkboxes. Each checkbox or radio button has its own `<label>`.

**Comment by SirCmpwn** [comment]
Removed help button, I didn't really like it being the only one with a help button anyway. Let's just document the whole page at once off of the header help button.

**Comment by eatonphil** [comment]
:+1:

File: test/data/linodes.js [hunk 2]
```diff
@@ -137,7 +137,25 @@ export const testLinode = {
       12345: {
         id: 12345,
         label: 'Test config',
-        // TODO: Fill this out more
+        comments: 'Test comments',
+        kernel: { id: 'linode/latest_64' },
```
Comment (na3d) [inline test/data/linodes.js]: Do you need kernel label or created here for anything testing?

Comment (SirCmpwn) [inline test/data/linodes.js]: Not yet, no.

Comment (na3d) [inline test/data/linodes.js]: Isn't the label used for the kernel dropdown?

Comment (SirCmpwn) [inline test/data/linodes.js]: It is, but that's not important for the tests.

**Comment by na3d** [comment]
👍

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**